tests/api/examples.spec.ts

```diff
@@ -394,6 +394,16 @@ describe("examples", () => {
         expect(testCaseFQ.expectedOutputQuery.trim()).equals(
           "select 'hi' as faked union all\nselect 'ben' as faked union all\nselect 'sup?' as faked"
         );
+
+        // Check double backslashes don't get converted to singular.
+        const exampleDoubleBackslash = graph.tables.find(
+          (t: dataform.ITable) =>
+            t.name === schemaWithSuffix("df_integration_test") + ".example_double_backslash"
+        );
+        expect(exampleDoubleBackslash).to.not.be.undefined;
+        expect(cleanSql(exampleDoubleBackslash.query)).equals(
```

Comment:
you should add an expectation on the value of `preOperations` (or whatever the field is called)

Reply:
Good idea. Turns out the SQL change didn't cover it!

Reply:
Broke the whole thing, added a separate state to the lexer to keep track of entering and leaving single and double quotes.